packages/backend/src/open_payments/receiver/model.ts

```diff
@@ -29,8 +29,15 @@ export class Receiver {
   public readonly assetCode: string
   public readonly assetScale: number
   public readonly incomingPayment: ReceiverIncomingPayment
+  public readonly isLocal: boolean
 
-  constructor(incomingPayment: OpenPaymentsIncomingPaymentWithPaymentMethod) {
+  constructor(
+    incomingPayment: OpenPaymentsIncomingPaymentWithPaymentMethod,
+    // TODO: lookup incomingPayment where isLocal is used instead of tracking isLocal here?
+    // not sure how simple the check can be. url matches resource server?
+    // exists in local db? exists in local db AND has stream creds (how its checked in receiverService.getReceiver)?
+    isLocal: boolean
```

Comment:
Going to remove this TODO and keep as-is.

I was trying to see if there was an easier way than passing this flag in everywhere we create a receiver (basically `receiverService.create` and `get` at the moment). However, we do have this information readily available in those places so I guess its not a big deal. I was thinking maybe we could infer it from the resourceServer but this is probably inferior to checking the actual existence of the payment in the local db.